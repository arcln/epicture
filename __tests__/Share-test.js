import 'react-native';
import React from 'react';
import Share from '../screens/ShareScreen';
import renderer from 'react-test-renderer';
import NavigationTestUtils from 'react-navigation/NavigationTestUtils';

describe('Share snapshot', () => {
  jest.useFakeTimers();
  beforeEach(() => {
    NavigationTestUtils.resetInternalState();
  });

  it('renders the screen', async () => {
    const tree = renderer.create(<Share />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
