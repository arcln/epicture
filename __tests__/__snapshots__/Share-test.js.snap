// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Share snapshot renders the screen 1`] = `null`;
